services/baseremotecontrol/builtin/builtin.go

```diff
@@ -125,28 +131,53 @@ func (svc *builtIn) start(ctx context.Context) error {
 	state := &throttleState{}
 	state.init()
 
-	var lastEvent input.Event
+	var lastTS time.Time
+	lastTSPerEvent := map[input.Control]map[input.EventType]time.Time{}
 	var onlyOneAtATime sync.Mutex
 
+	updateLastEvent := func(event input.Event) bool {
```

Comment:
we were dropping events too often for unrelated controls. this allows us to be more specific about accepting events for a multi control situation. it also supports being able to heartbeat through AllEvents, if we wanted that